src/scss/utilities/mixins/_top-banner.scss

```diff
@@ -18,7 +18,9 @@
   background: $su-color-bright-red;
 
 
-  .su-button {
```

Comment:
Let's add a comment here that says we are enforcing that only the secondary button style can be used on this red background.